### DISCUSSION TITLE
Basic Server Example with Next.js

### DISCUSSION THREAD

**Discussion by targetlucked69** [connectrpc/connect-es#360] [Q&A] [answered]
Hi! I have tried the [Nextjs](https://github.com/bufbuild/connect-web-integration/tree/main/nextjs) example and it worked pretty great. 

Is there a simple example somewhere of the server?  Where https://demo.connect.build is running? 

Can Nextjs APIs be a server? Thanks!

**Comment by timostamm** [comment, marked as answer]
Hey Jeremy, the example server is written in Go. You can see the source [here](https://github.com/bufbuild/connect-demo). 
We are currently working on a Node.js implementation, which is still a couple of weeks out. It simply creates standard Node.js request handlers. In general, the handlers are compatible with Next.js API routes, but there are still a couple of Gotchas and kinks that we'd want to address before we call Next.js supported.
See also https://github.com/bufbuild/connect-web/discussions/344
